homeassistant/components/group.py

```diff
@@ -112,6 +113,11 @@ def setup(hass, config):
 
 class Group(object):
```

Comment:
I think we should move to have Group extend from Entity in such a way that we do not have to duplicate the visibility logic here.